services/movie.py

```diff
@@ -0,0 +1,40 @@
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(
+            genres__id__in=genres_ids, actors__id__in=actors_ids
+        )
+    elif genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    elif actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
```

Comment:
This line is correct. The `.get()` method is used appropriately to retrieve a single `Movie` object by its ID.